app/components/Poll/Poll.css

```
@@ -1,16 +1,83 @@
 @import url('~app/styles/variables.css');
 
 .poll {
-  min-height: 160px;
+  background-color: var(--lego-card-color);
+  width: 100%;
+  min-width: 250px;
+  max-width: 500px;
+  min-height: 120px;
+  margin: 0 auto;
+  overflow-y: hidden;
+  border-radius: 5px;
+}
+
+.topBar {
+  width: 100%;
+  z-index: 2;
+}
+
+.pollIcon {
+  display: block;
+  width: 100%;
+  text-align: center;
+  background-color: var(--lego-red-color);
+  color: var(--lego-card-color);
+  padding: 0 0.5rem 0.5rem;
+  box-shadow: 0 2px 2px rgba(0, 0, 0, 20%);
+}
+
+.titleLink {
+  position: relative;
+  display: block;
+  width: 80%;
+  top: -1.25rem;
+  margin-bottom: -1rem;
+  border-radius: 10px;
+  color: var(--lego-font-color);
+  background-color: var(--lego-card-color);
+  box-shadow: 0 3px 3px rgba(0, 0, 0, 20%);
+  z-index: 3;
+}
+
+.title {
+  padding: 0.5rem;
+  font-weight: 500;
+  text-align: center;
+  line-height: 1.5;
+}
+
+.contentWrapper {
+  width: 100%;
+  transition: height 0.5s;
+  overflow-y: hidden;
 }
 
 .optionWrapper {
+  display: flex;
+  width: 100%;
   justify-content: center;
-  margin-top: 4px;
-  min-height: 120px;
-  position: relative;
+  align-items: center;
+  margin: 0.5rem 0;
+}
+
+.description {
+  width: 90%;
+  padding-bottom: 0.5rem;
 }
```

```suggestion
.description {
  width: 90%;
  padding-bottom: 0.5rem;
  color: var(--secondary-font-color);
}
```

Just a suggestion - haven't checked if it looks any good